be/src/vec/exec/format/parquet/vparquet_column_reader.h

```diff
@@ -262,23 +263,24 @@ class StructColumnReader : public ParquetColumnReader {
             : ParquetColumnReader(row_ranges, ctz, io_ctx) {}
     ~StructColumnReader() override { close(); }
 
-    Status init(std::vector<std::unique_ptr<ParquetColumnReader>>&& child_readers,
-                FieldSchema* field);
+    Status init(
+            std::unordered_map<std::string, std::unique_ptr<ParquetColumnReader>>&& child_readers,
+            FieldSchema* field);
     Status read_column_data(ColumnPtr& doris_column, DataTypePtr& type,
                             ColumnSelectVector& select_vector, size_t batch_size, size_t* read_rows,
                             bool* eof, bool is_dict_filter) override;
 
     const std::vector<level_t>& get_rep_level() const override {
-        return _child_readers[0]->get_rep_level();
+        return _child_readers.begin()->second->get_rep_level();
```

Comment:
Only the read fields can return right levels, the missing fields will return empty level value.

Reply:
So the tow level nested type like `struct<struct<>>` will be parsed badly.